### ISSUE TITLE
Can not navigate to project view in issue page 

### ISSUE THREAD

**Issue by georgexsh** [CodeHubApp/CodeHub#444]
As there is no link in issue page, if I access a issue from a notification, I simply can not go the project own the issue. Currently issue page show the user profile icon who opened the issue, I guess project info is more suitable.

**Comment by garrettw** [comment]
I agree that this is needed. I've wanted this feature on many occasions.

**Comment by georgexsh** [comment]
besides, there is no edit feature for issue body and comment.

**Comment by georgexsh** [comment]
ironically, I want visit repo of codehub from this issue, but failed.
